android/src/main/java/com/revenuecat/purchases/react/PaywallFooterViewManager.java

```diff
@@ -0,0 +1,24 @@
+package com.revenuecat.purchases.react;
+
+import android.annotation.SuppressLint;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.uimanager.SimpleViewManager;
+import com.facebook.react.uimanager.ThemedReactContext;
+import com.revenuecat.purchases.ui.revenuecatui.views.PaywallFooterView;
+
+public class PaywallFooterViewManager extends SimpleViewManager<PaywallFooterView> {
+    @NonNull
+    @Override
+    public String getName() {
+        return "RCPaywallFooterView";
+    }
+
+    @SuppressLint("UnsafeOptInUsageError")
+    @NonNull
+    @Override
+    protected PaywallFooterView createViewInstance(@NonNull ThemedReactContext themedReactContext) {
+        return new PaywallFooterView(themedReactContext);
```

Comment:
We will still need to think about how other parameters from the RN layer to the native view and back (for the listeners).